### PR TITLE
PF Graph performance tests.

### PR DESCRIPTION
**Pull request by hhovsepy**
### Describe the change

Adjusted existing Graph performance tests to PF Graph.
Taking 3 measurements:

1. All Namespaces.
2. All Namespaces with Idle nodes.
3. Selected (few) Namespaces.

### Automation testing

Perf tests added

### Issue reference
https://github.com/kiali/kiali/issues/7709

### CODE REVIEW

File: frontend/cypress/perf/common.ts [hunk 1]
```diff
@@ -33,7 +39,12 @@ const measureLoadTime = (name: string, baseline: number, loadUrl: string, loadEl
           cy.on('uncaught:exception', (err, runnable) => {
             return false;
           });
-          cy.get(loadElementToCheck).should('be.visible');
+          if (isGraph) {
+            cy.waitForReact();
+            cy.getReact('GraphPF').should('have.length', '1');
```
Comment (ferhoyos) [inline frontend/cypress/perf/common.ts:44]: I think we should double check here that the Graph is really loaded (maybe the component `GraphPF` exists when the graph is being loaded). I would add an extra assertion here, e.g., the graph nodes length is more than 0 or the message `Graph Loading` is not displayed. Look at https://github.com/kiali/kiali/pull/7799 for some examples.

**Review by ferhoyos — APPROVED** [review]
LGTM. Thanks @hhovsepy for adding more assertions to the graph load. Checking that nodes and edges exist will ensure the graph has been loaded correctly.

File: frontend/cypress/perf/common.ts (hunk 1 above)
Comment (hhovsepy) [reply frontend/cypress/perf/common.ts]: have changed to `cy.getReact('GraphPagePFComponent', { state: { isReady: true } }).should('have.length', '1');`, the double check with `graphRefs` causes Jenkins job to fail with CPU/Memory issues.
for now having this solution with TODO

**Review by ferhoyos — APPROVED** [review]
LGTM

**Pull request merged**